_data/providers.json

```
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Use the 19132 port and the server ip and connect with the same IP and port 19132. You can also ask for a new port."
```

19132 is the default port anyways, so users don't have to specify it..

```suggestion
      "description": "Use the default bedrock port (19132) and the server ip to connect on Bedrock edition. No changes to the Geyser config are needed. You can also ask for a new port and use it instead."
```